blocks/floating-images/floating-images.css

```diff
@@ -43,14 +55,32 @@
   line-height: var(--line-height-m);
 }
 
-.block.floating-images p.button-container {
-  text-align: center;
+.block.floating-images p .button {
```

Comment:
Can you check this and compare it to the use of `p.button-container a` and/or `p.button-container a.primary` throughout the site, if we can just update those, that'd be better.

Reply:
Took a look and it seems like this is the main primary button style of the new design, so I moved those button styles to the main styles.css file.